desktop/core/requirements.txt

```diff
@@ -9,7 +9,7 @@ channels-redis==2.3.2
 configobj==5.0.6
 cryptography==3.2
 django-auth-ldap==1.3.0
-Django==1.11.29 # Django 2 then 3?
+Django==2.0.13 # Django 2 then 3?
```

Comment:
How about we split the PR in two? (to not mix and match)

We do this line change and the failing Hive tests in a follow-up and can focus there?